src/modules/educations/educations.service.ts

```diff
@@ -22,10 +23,9 @@ export class EducationsService {
     req: Request
   ) : Promise<Education> {
     const { Title, Certificate_Place, Level, Graduation_Time } = createEducationInput;
-    const userId: string = this.utilsService.getUserIdFromHeader(req);
-    const profile: Profile = await this.profileService.getProfileByUserId(userId);
+    const profile: Profile = await this.profileService.getProfileFromHeader(req);
 
-    const newEducation: Education = new Education();
+    const newEducation: Education = this.educationRepository.create();
     newEducation.Title = Title;
     newEducation.Certificate_Place = Certificate_Place;
     newEducation.Level = Level;
```

Comment:
const newEducation: Education = this.educationRepository.create({
 Title, Certificate_Place, Level
});